exercises/simple-linked-list/Example.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+public class SimpleLinkedList<T>
+{
+    public SimpleLinkedList(T value) :
+        this(value, null)
+    {
+    }
+
+    public SimpleLinkedList(T value, SimpleLinkedList<T> next)
+    {
+        Value = value;
+        Next = next;
+    }
+
+    public T Value { get; }
+
+    public SimpleLinkedList<T> Next { get; }
+
+    public static SimpleLinkedList<T> FromEnumerable(IEnumerable<T> values)
```

Comment:
I did some modifications to better align this implementation with that of the binary-search-tree exercise. As such, the singleton methods are replaced by constructors, I've implemented IEnumerable<T> and I have added an `Add()` method that I use in the class itself to add new items to the collection.